pjdfstest.c

```diff
@@ -96,8 +96,10 @@
 #define	HAS_FSTATAT
 #define	HAS_LINKAT
 #define	HAS_MKDIRAT
+#ifndef	__APPLE__
```

Comment:
I didn't want to be too invasive here with autoconf. I'll do it in the next stage.